### PR TITLE
[tests] Test listwallets RPC

### PR DESCRIPTION
**Pull request by mess110**

### CODE REVIEW

**Review by promag — CHANGES_REQUESTED** [review]

File: test/functional/wallet-accounts.py [hunk 1]
```diff
@@ -78,54 +81,54 @@ def run_test(self):
         for account in accounts:
             address = node.getaccountaddress(account)
             account_addresses[account] = address
-            
+
```
[the remark this answers is not in the document]
Comment (mess110) [reply test/functional/wallet-accounts.py]: Was thinking of the boy scout rule, but yes, it is unrelated to the patch

**Comment by jnewbery** [comment]
I have a test for `listwallets` in #10740 in multiwallet.py, which I think makes more sense than adding to wallet-accounts.py. The commit is here: https://github.com/bitcoin/bitcoin/pull/10740/commits/915c65a3bd6b7ace49e7046d78574390d2302e6f

I suggest you either close this PR or change it to add the `listwallets` test to multiwallet.py.

**Comment by mess110** [comment]
I moved the code to `multiwallet.py`.

I would close this PR if we know #10740 will be merged. The feature is clearly helpful, and will probably be merged.

Taking this into account, if you think this patch is still useful, please continue the review process, otherwise please close it.

**Comment by promag** [comment]
Easy utACK d510ec5.

**Comment by jnewbery** [comment]
#10740 requires a lot of review and test, so I think it's fine to merge this first.

My only comment is that `listwallets` doesn't necessarily guarantee an order for the returned wallets, so perhaps converting to a set and doing a set comparison is more correct.

However, I'm probably just being overly picky. utACK d510ec5dff4de1df03463dda4238da3ceb60f8f3

**Pull request merged**